### PR TITLE
feat(website): create MentorSchedule section

### LINKED ISSUE #107: Create MentorSchedule section
**Mockups:**
[Figma link](https://www.figma.com/file/ur0KLA9ZOtiodrAmpXrxh9/CodersCamp-app-and-website?node-id=829%3A2778)

### PR DESCRIPTION
**Pull request by talcia**
Closes #107

### CODE REVIEW

File: packages/website/src/components/Schedule/ScheduleContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { Typography } from '@coderscamp/ui/components/Typography';
+
+interface ScheduleContainerProps {
+  title: string;
+  children: ReactElement;
+}
+
+export const ScheduleContainer = ({ title, children }: ScheduleContainerProps) => {
+  return (
+    <Center flexDirection="column" mx="auto" w="100%" mt="120px" mb="80px">
```
Comment (KonradSzwarc) [inline packages/website/src/components/Schedule/ScheduleContainer.tsx:13]: 🤔 Total margin-top and margin-bottom of this section should be 80px each. You can remove `mt` and `mb` here and replace them with `my="80px"`. Additionaly you will need to change `m="64px 0"` below to `mb="64px"`

File: packages/website/src/components/Schedule/ScheduleContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      <Typography size="4xl" weight="extrabold" m="64px 0" textAlign="center">
```
Comment (KonradSzwarc) [inline packages/website/src/components/Schedule/ScheduleContainer.tsx:14]: 🤔 To match mockups add `color="gray.900"`

File: packages/website/src/components/Schedule/MentorSchedule/MentorSchedule.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  {
+    index: 4,
+    date: '01.10.2021',
+    description: 'Otrzymasz od nas informację zwrotną o wyniku rekrutacji.',
+  },
+];
+
+export const MentorSchedule = () => {
+  return (
+    <ScheduleContainer title="Chce zostać mentorem - co dalej?">
```
Comment (KonradSzwarc) [inline packages/website/src/components/Schedule/MentorSchedule/MentorSchedule.tsx:42]: 🙃 Chce -> Chcę

**Review by KonradSzwarc — CHANGES_REQUESTED** [review]

**Review by KonradSzwarc — APPROVED** [review]

**Review by KonradSzwarc — APPROVED** [review]

File: packages/website/src/pages/mentor.tsx [hunk 4]
```diff
@@ -1,11 +1,13 @@
 import { MentorPerspective } from '@/components/MentorPerspective';
 import { MentorRequirements } from '@/components/MentorRequirements';
+import { MentorSchedule } from '@/components/Schedule/MentorSchedule/MentorSchedule';
 
 const Mentor = () => {
   return (
     <>
-      <MentorPerspective />
       <MentorRequirements />
+      <MentorPerspective />
```
Comment (KrystianKjjk) [inline packages/website/src/pages/mentor.tsx:9]: Why MentorPerspective is moved under MentorRequirements ?

Comment (talcia) [reply packages/website/src/pages/mentor.tsx:9]: Ups sorry my bad

**Review by lagdush — APPROVED** [review]

**Review by wierzbka10 — APPROVED** [review]

**Pull request merged**